app/scripts/services/projects.js

```diff
@@ -23,6 +23,7 @@ angular.module('openshiftConsole')
           return  AuthService
                     .withUser()
                     .then(function() {
+                      AuthorizationService.getProjectRules(projectName);
```

Comment:
Tend to agree this should be in a promise chain. Might be able to use `$q.all([])` to resolve in parallel, but that could be a pain for updating calling code.  

I'd tend to think line 40 could be:
```javascript
 return [project, context, projectRules];
```
and having a direct handle on a projectRules object for the particular project would be handy.